e2e/e2e-tests-constants.js

```diff
@@ -0,0 +1,5 @@
+import advisories from '../src/advisories.js';
+
+export const NUM_OF_ADVISORIES = Object.keys(advisories).length;
+export const PACKAGE_REPORT_SELECTOR = 'overlay-package-report';
```

Comment:
Sure, I can create a PR with this change. 
Just to make sure - do you want that both constants: `PACKAGE_REPORT_SELECTOR` and `NUM_OF_ADVISORIES` will be placed in another file, or just `NUM_OF_ADVISORIES`?

Reply:
Only `PACKAGE_REPORT_SELECTOR`, and it should be called something like `packageReportTagName` and be used everywhere instead of a string literal.

If you will do it in another PR, I will merge this right now.

You can reference this comment in your new PR, no need to open a new issue.